docs/reach/common/reachview/firmware-reflashing.md

```diff
@@ -31,88 +31,71 @@ Note that you **do not** need to do this unless you want to bring Reach to its i
 	
 	|Windows|Linux|
 	|:-------------:|:----------:|
-	|[Download [EXE, 79.9 MB]](http://files.emlid.com/flash-tools/win/reach-firmware-flash-tool_1.1.3_setup.exe)|[Download [DEB, 53.9 MB]](http://files.emlid.com/flash-tools/linux/reach-firmware-flash-tool_1.1.3_amd64.deb)|
+	|[Download [EXE, 81.9 MB]](http://files.emlid.com/flash-tools/win/reach-firmware-flash-tool_1.2.0_setup.exe)|[Download [DEB, 109.8 MB]](http://files.emlid.com/flash-tools/linux/reach-firmware-flash-tool_1.2.0_amd64.deb)|
 
 	</center>
 
 	* Double click on the downloaded file
 	
 	* Follow the instructions to install Reach Firmware Flash Tool on your PC
 
+	!!! danger "" 
+		After installing, it is required to reboot your PC
+
 	### Flashing process
 
 	* Open Reach Firmware Flash Tool and choose Reach model
 
 	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step1-device-select.png" style="width: 300px;" /></p>
 
-	<!-- delete part with RS -->
-
-	* Select ReachView firmware .img file
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step2-image-select.png" style="width: 300px;" /></p>
-
-	* Disconnect all Reach units. If there are none, Flash Tool will automatically skip this step
-	
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step3-disconnect-devices.png" style="width: 300px;" /></p>
-
-	* Connect Reach device and turn it on. "Reach connected" message will appear
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step4-connected-reach.png" style="width: 300px;" /></p>
-
-	* Wait until the end of the flashing process
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step5-flashing.png" style="width: 300px;" /></p>
-
-	* After reflashing, Reach will reboot
-
-	!!! danger ""
-		Do not unplug the Reach on this step.
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step5-rebooting.png" style="width: 300px;" /></p>
-
-	* When Reach successfully reboots, you'll see "Reach is flashed and ready to use" message
-
-	<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-step5-flashed-reach.png" style="width: 300px;" /></p>
-
-	??? danger ""Flashing failed" error troubleshooting"
-
-		If you faced "Flashing failed" error, we recommend following the next steps:
-
-		<p style="text-align:center" ><img src="../img/reachview/firmware-reflashing/app-flashing-failed.png" style="width: 300px;" /></p>
+	* Connect Reach in Firmware mode
 
 		??? note "Steps for RS2 device"
 			
-			* Disconnect **Reach RS2** device from the PC
+			* Turn off the receiver using the power button
 			
-			* Hold the power button for 10 seconds until Reach turns off
+			* Press and hold the power button again and while pressing plug the USB into PC
```

Comment:
@TatianaAndreeva is this step clear for you? I suggest splitting it into 3 separate steps: press and hold the button, plug the USB cable, release the button.

Reply:
I agree